platform/android/export/export.cpp

```diff
@@ -2274,92 +2274,103 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		return have_plugins_changed || first_build;
 	}
 
-	virtual Error export_project(const Ref<EditorExportPreset> &p_preset, bool p_debug, const String &p_path, int p_flags = 0) {
-
-		ExportNotifier notifier(*this, p_preset, p_debug, p_path, p_flags);
-
-		String src_apk;
-
-		EditorProgress ep("export", "Exporting for Android", 105, true);
 
-		if (bool(p_preset->get("custom_template/use_custom_build"))) { //custom build
-			//re-generate build.gradle and AndroidManifest.xml
+	Error gradle_build(const Ref<EditorExportPreset> &p_preset, bool p_debug, const String &p_path, String &src_apk, int p_flags = 0){
```

Comment:
I'm not sure it applies since this method is unique to this class only, and it isn't called outside of this class: https://www.geeksforgeeks.org/virtual-function-cpp/

Reply:
Ack, I wasn't sure if it would be. No change needed.